root/static/scripts/release-editor/edits.js

```diff
@@ -78,7 +78,7 @@ releaseEditor.edits = {
             edits.push(MB.edit.releaseCreate(newData));
         }
         else if (!_.isEqual(newData, oldData)) {
-            newData = _.extend(_.clone(newData), { to_edit: release.gid() });
+            newData = Object.assign({}, newData, { to_edit: release.gid() });
```

Comment:
Instead of using `_.clone`, I made it assign to an empty object, which has the same effect (`_.clone` creates a shallow clone).